frontend/pages/ManageControlsPage/SetupExperience/cards/Users/_styles.scss

```
@@ -1,3 +1,8 @@
 .setup-experience-users {
   @include vertical-card-layout;
+  gap: $gap-page-component-inner;
```

Stylelint (17.10.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)